chainerx/_docs/routines.py

```diff
@@ -417,6 +417,30 @@ def _docs_creation():
 .. seealso:: :func:`numpy.diagflat`
 """)
 
+    _docs.set_doc(
+        chainerx.meshgrid,
+        """meshgrid(arrays, indexing="xy")
+Return coordinate matrices from coordinate vectors.
```

Comment:
Ditto (although NumPy says "Return").
```suggestion
Returns coordinate matrices from coordinate vectors.
```

Reply:
Done.